wikicharts/content_gap_plots.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #61.        plt.annotate(f"Average: {mean_y:.1%}",
</pre><p>How hard do you think it would be add this an <code>annotate_average</code> function to Wikicharts? It would save on duplicated code. Not a blocker to this merge, but worth doing now if it would be easy.</p><p> </p><p>Separately, would it be hard to add a horizontal line indicating the average? Seems like that would be a useful addition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>